integration-tests/eks/resourceCount_windowslinux_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build windowslinux
+// +build windowslinux
+
+package eks_addon
+
+const (
+	serviceCountLinux     = 4
+	serviceCountWindows   = 3
```

Comment:
Can we have a comment in the test file why these values are different in this case

Reply:
Add comments